src/components/Navigation/TopBar/Notifications/NotificationsTemplate/NotificationElements/NoNotifications.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+
+import { Typography } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+
+import { FeedBackIcon } from '../../../../../index';
```

Comment:
Really long import here, without looking at the tree I think it can be shortened?